baselines/dom.generated.d.ts

```diff
@@ -1507,11 +1524,26 @@ interface UIEventInit extends EventInit {
     view?: Window | null;
 }
 
-interface UnderlyingSink {
-    abort?: WritableStreamErrorCallback;
-    close?: WritableStreamDefaultControllerCallback;
-    start: WritableStreamDefaultControllerCallback;
-    write?: WritableStreamChunkCallback;
+interface UnderlyingByteSource {
+    autoAllocateChunkSize?: number | null;
+    cancel?: ReadableStreamErrorCallback | null;
+    pull?: ReadableByteStreamControllerCallback | null;
+    start?: ReadableByteStreamControllerCallback | null;
+    type: "bytes";
+}
+
+interface UnderlyingSink<W = any> {
+    abort?: WritableStreamErrorCallback | null;
+    close?: WritableStreamDefaultControllerCallback | null;
+    start?: WritableStreamDefaultControllerCallback | null;
+    write?: WritableStreamDefaultControllerWriteCallback<W> | null;
+}
+
+interface UnderlyingSource<R = any> {
+    cancel?: ReadableStreamErrorCallback | null;
+    pull?: ReadableStreamDefaultControllerCallback<R> | null;
+    start?: ReadableStreamDefaultControllerCallback<R> | null;
+    type?: undefined;
```

Comment:
This looks a bit weird, but it is intentional. [The spec says](https://streams.spec.whatwg.org/#rs-constructor):

> 6. If _typeString_ is `"bytes"`,
> 7. Otherwise, if _type_ is `undefined`,
> 8. Otherwise, throw a `RangeError` exception.

With this type definitions, we can catch errors like this:
```
var readable = new ReadableStream({
    type: 'bytez', // oops!
    start(c) {
        c.enqueue(new Uint8Array([1]));
    }
});
```